### DISCUSSION TITLE
Transfered-In Domain Not Visible in Vercel Dashboard

### DISCUSSION THREAD

**Discussion by sean1832** [vercel/community#6772] [Help] [answered]
### Summary

I'm new to using Vercel and recently transferred a domain from Hostinger to Vercel few weeks ago, for which I paid $20. After the transfer, the domain was successfully disconnected from Hostinger. However, I'm facing an issue where the transferred domain does not appear under `Projects > Domains` in my Vercel dashboard.

Interestingly, I was able to assign the transferred domain to my projects without any issues. Despite this, I'm concerned about not being able to view or manage the domain directly from the Vercel dashboard and that I may loose control over this domain.

Could anyone help me understand why this might be happening or if I'm missing a step in the domain management process on Vercel? Any guidance or advice would be greatly appreciated!

Thank you!

![image](https://github.com/vercel/community/assets/84948038/7c098956-1f49-4089-853f-50ca0d74a49d)

### Steps to Reproduce

1. Transfer Domain
2. Entered the domain to transfer and authorization code
3. Unlock domain in Hostinger
4. Wait for few weeks, the domain did not show up under `projects>Domains` but can connect to projects just fine.

**Comment by amyegan** [comment, marked as answer]
Hey @sean1832. The domain may be associated with your _account_ rather than your _team_. You should see it listed at [vercel.com/account/domains](https://vercel.com/account/domains), and you can [move your domain](https://vercel.com/docs/projects/domains/transfer-your-domain) to the team if you'd like it manage it there.

**Reply by sean1832** [reply]
Ah, that's it. I have found it under the `Account>Domains`. Thank you so much fo the reply. I was so confused for the past few weeks.
